Library/Homebrew/cmd/upgrade.rb

```diff
@@ -41,7 +42,10 @@ def upgrade
       puts pinned.map { |f| "#{f.full_name} #{f.pkg_version}" } * ", "
     end
 
-    outdated.each { |f| upgrade_formula(f) }
+    outdated.each do |f|
+      upgrade_formula(f)
```

Comment:
@xu-cheng I had missed your point earlier, I get it know. 👍🏽 I'll update the commit with your suggestion. Thanks for the reviews the code .